src/assets/styles/includes.scss

```diff
@@ -7,6 +7,7 @@
 @import "host";
 @import "spacing";
 @import "popper";
+@import "hydration";
```

Comment:
Should this import go in `globals.scss` or are they here so that they get bundled with each component (e.g., so styling applies when wrapped in other components)? Currently, it looks like the styles are built into both the global CSS file and each component.

Reply:
no, it should only be needed in the global file, good catch